pegasus/router.rb

```diff
@@ -110,9 +110,12 @@ def self.set_max_age(type, default)
       settings.template_extnames +
       settings.exclude_extnames +
       ['.fetch']
+    # Note: shared_resources.rb has additional configuration for Sass::Plugin
     Sass::Plugin.options[:cache_location] = pegasus_dir('cache', '.sass-cache')
-    Sass::Plugin.options[:css_location] = pegasus_dir('cache', 'css')
-    Sass::Plugin.options[:template_location] = shared_dir('css')
```

Comment:
Removed this redundant configuration for /shared/css in Pegasus.

Reply:
The "official" documentation appears to be no longer available, but here's a fork: http://paulirish.github.io/Sass-reference/.  Very briefly, what the gem does is looks for sass files in `css_location`, compiles them, and writes the output to `template_location`.  I've replaced the existing way of configuration them in options (which allows for a single mapping) to calling `add_template_location` which allows for configuring multiple mappings.

We used to configure this particular mapping (pegasus/cache/css --> shared/css) redundantly in pegasus and in shared, possibly because of some [pegasus tests issues](https://github.com/code-dot-org/code-dot-org/pull/16040).  This mapping is now only configured in `shared_resources.rb` and the new pegasus-specific sass directory mapping is configured here.  I've attempted to fix the test issue a different way (see change in `test_pegasus_documents.rb`) and will keep an eye if that's sufficient.